tests/test_server.py

```diff
@@ -560,8 +539,61 @@ def test_pandadf_dump_flat(self):
         self.assertTrue("post_mean" in out_df.columns)
         self.assertTrue("post_var" in out_df.columns)
 
+    """
+    Tests that test the server connections, as well as message routing
+    """
+
+    def test_receive(self):
+        """test_receive - verifies the receive is working when server receives unexpected messages"""
+
+        message1 = b"\x16\x03\x01\x00\xaf\x01\x00\x00\xab\x03\x03\xa9\x80\xcc"  # invalid message
+        message2 = b"\xec\xec\x14M\xfb\xbd\xac\xe7jF\xbe\xf9\x9bM\x92\x15b\xb5"  # invalid message
+        message3 = {"message": {"target": "test request"}}  # valid message
+        message_list = [message1, message2, json.dumps(message3)]
+
+        i = 0
+        self.s.socket.conn = MagicMock()
+
+        for message in message_list:
```

Comment:
You can use `for i, message in enumerate(message_list)` so that you don't have to manually keep track of a separate `i` index.

Reply:
Correct, forgot to enumerate here to save some space.